opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/StixCoreRelationshipCreationFromEntityStixCoreObjectsLine.jsx

```diff
@@ -139,195 +139,197 @@ const StixCoreRelationshipCreationFromEntityStixCoreObjectsLineComponent = ({
   );
 };
 
+export const stixCoreRelationshipCreationFromEntityStixCoreObjectsLineFragment = graphql`
+  fragment StixCoreRelationshipCreationFromEntityStixCoreObjectsLine_node on StixCoreObject {
+    id
+    standard_id
+    parent_types
+    entity_type
+    created_at
+    ... on AttackPattern {
+      name
```

Comment:
I wonder if we could simplify this fragment by using representatives

Reply:
useToggleEntity and shift entity are based on name
I risk breaking a lot of things if I change to this exact location